charts/posthog/templates/events-deployment.yaml

```diff
@@ -14,6 +14,13 @@ spec:
   {{- if not .Values.events.hpa.enabled }}
   replicas: {{ .Values.events.replicacount }}
   {{- end }}
+
```

Comment:
Do we want to change events only? What about the others?

Reply:
I'm only really thinking of configuring those two atm, but I guess we could add this to everything 🤔 Will add that now

Reply:
Ah 👍 . I saw the changes in `values.yaml` and I thought some files were missing from the git commit.

Reply:
Yeah the plugins deployment is all fancy and templated, so it doesn't need as many file changes